### PR TITLE
Gdr 2419

### PR DESCRIPTION
**Pull request by gladkia**
# Description
Improve pkgdown site
## What changed?
- updated news and added reference sections
- Related JIRA issue:  GDR-2419
- pkgdown site link: https://gdrplatform.github.io/gDRcore/

# Logistic checklist
- [X] Package version bumped
- [X] Changelog updated

# Screenshots (optional)

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,532 +1,537 @@
...
 
-## 0.1.3.4 (2022-09-29)
-- Change the logic for using cores in `BiocParallel`
+## gDRcore 0.1.3.5 - 2022-10-07
+* remove global parameters for `BiocParallel`
 
-## 0.1.3.3 (2022-09-27)
-- update the logic for parallel computing
+## gDRcore 0.1.3.4 - 2022-09-29
+* Change the logic for using cores in `BiocParallel`
```
Comment (j-smola) [inline NEWS.md:245]:
```suggestion
* change the logic for using cores in `BiocParallel`
```

File: NEWS.md [hunk 2]
```diff
@@ -1,532 +1,537 @@
...
 
-## 0.1.3.11 (2022-12-16)
-- Replace parallelize function with gDRutils::loop
+## gDRcore 0.1.3.12 - 2023-01-12
+* refactor mapping function to properly handle drug3
 
-## 0.1.3.10 (2022-12-15)
-- add assert for vehicle values in input data in `runDrugResponseProcessingPipeline`
+## gDRcore 0.1.3.11 - 2022-12-16
+* Replace parallelize function with gDRutils::loop
```
Comment (j-smola) [inline NEWS.md:224]:
```suggestion
* replace parallelize function with gDRutils::loop
```

File: NEWS.md [hunk 3]
```diff
@@ -1,532 +1,537 @@
...
 
-## 0.1.3.13 (2023-01-13)
-- Clean-up code
+## gDRcore 0.1.3.14 - 2023-02-10
+* Clean-up code
 
-## 0.1.3.12 (2023-01-12)
-- refactor mapping function to properly handle drug3
+## gDRcore 0.1.3.13 - 2023-01-13
+* Clean-up code
```
Comment (j-smola) [inline NEWS.md:218]:
```suggestion
* clean-up code
```

File: NEWS.md [hunk 4]
```diff
@@ -1,532 +1,537 @@
...
 
-## 0.1.3.14 (2023-02-10)
-- Clean-up code
+## gDRcore 0.1.3.15 - 2023-02-10
+* update path to annotation data
 
-## 0.1.3.13 (2023-01-13)
-- Clean-up code
+## gDRcore 0.1.3.14 - 2023-02-10
+* Clean-up code
```
Comment (j-smola) [inline NEWS.md:215]:
```suggestion
* clean-up code
```

File: NEWS.md [hunk 5]
```diff
@@ -1,532 +1,537 @@
...
 
-## 0.99.19 (2023-06-06)
-- replaced reshape2 functions by functions from data.table
+## gDRcore 0.99.20 - 2023-06-07
+* switch from `zoo::rollmean` to `data.table::frollmean`
 
-## 0.99.18 (2023-05-31)
-- fix managing of mixed types of raw data
+## gDRcore 0.99.19 - 2023-06-06
+* replaced reshape2 functions by functions from data.table
```
Comment (j-smola) [inline NEWS.md:128]:
```suggestion
* replace reshape2 functions by functions from data.table
```

File: NEWS.md [hunk 6]
```diff
@@ -1,532 +1,537 @@
...
 
-## 0.99.32 (2023-07-25)
-- extended logic for supporting cols with dash, e.g. additional perturbations with "-"
+## gDRcore 0.99.33 - 2023-08-10
+* update annotation column names for cell line annotation as per changes in the gDRutils
 
-## 0.99.31 (2023-07-19)
-- update the logic for handling warnings in the pipeline
+## gDRcore 0.99.32 - 2023-07-25
+* extended logic for supporting cols with dash, e.g. additional perturbations with "-"
```
Comment (j-smola) [inline NEWS.md:88]:
```suggestion
* extend logic for supporting cols with dash, e.g. additional perturbations with "-"
```

**Review by j-smola — COMMENTED** [review]
Could you maybe rethink group for functions?
1. IMO not all function in "Combination helpers" is used only for combination data.
2. "Mappping" seems to be list of function supporting "Pipeline"
3. "Utils" and "Preparing input" are both "Preparing input for pipline"? As also "Identifying keys", right?
4. etc.



Please, consider also
1. update or "gDRcore.Rmd" vignette - "Drug processing section" section where due to lack of empty line list of steps disappeared;
5. update the gDRcore architecture diagram in READMA.md as it is no longer readable on the pkgdown site.

PS: And fot is the reasone for function from `constant.R` file?

**Comment by gladkia** [comment]
> Could you maybe rethink group for functions?
> 
> 1. IMO not all function in "Combination helpers" is used only for combination data.
> 2. "Mappping" seems to be list of function supporting "Pipeline"
> 3. "Utils" and "Preparing input" are both "Preparing input for pipline"? As also "Identifying keys", right?
> 4. etc.
> 
> Please, consider also
> 
> 1. update or "gDRcore.Rmd" vignette - "Drug processing section" section where due to lack of empty line list of steps disappeared;
> 2. update the gDRcore architecture diagram in READMA.md as it is no longer readable on the pkgdown site.
> 
> PS: And fot is the reasone for function from `constant.R` file?

# 1st section
1. Thanks. Fixed.
2. Yes. Please note that `runDrugResponseProcessingPipeline()` is the top-level function for running the whole pipeline, dependent on multiple functions from gDRcore.
3. `Prepare input` refers to the S3 methods called 'prepare_input' for data.table and MAE as well as their dependencies (internal helper functions). The `utils` comprises a broader, less specific category  I've moved a single function from 'Identifying keys' to the utils category. 

# 2nd section
1. Done.
2. Good idea. I'll try to prepare an analogous diagram with plantuml.
3. 'constants.R' - there are no functions there - only three `constant` objects: `drugNameRegex`, `untreated_tag_patterns`, and `untreatedDrugNameRegex`.

File: README.md [hunk 7]
```diff
@@ -1,58 +1,7 @@
-## Architecture
+## gDRcore
```
Comment (j-smola) [inline README.md:1]:
```suggestion
# gDRcore
```

File: vignettes/gDRcore.Rmd [hunk 8]
```diff
@@ -39,14 +39,21 @@ a series or the same number of elements, the attributes (columns of the table) s
 ## Drug processing 
 For drug response data, the input files need to be merged such that each measurement (data) is associated with the right metadata (cell line properties and treatment definition). Metadata can be added with the function `cleanup_metadata` if the right reference databases are in place.
 
-When the data and metadata are merged into a long table, the wrapper function `runDrugResponseProcessingPipeline` can be used to generate an MAE with processed and analyzed data. In practice runDrugResponseProcessingPipeline does the following steps:
```
Comment (j-smola) [inline vignettes/gDRcore.Rmd:42]: Maybe also this chunk wit loading libraries could be with `echo=FALSE`?

**Review by j-smola — APPROVED** [review]

File: vignettes/gDRcore.Rmd (hunk 8 above)
Comment (gladkia) [reply vignettes/gDRcore.Rmd:42]: done.

**Pull request merged**